src/util_i_debug.nss

```diff
@@ -264,7 +264,14 @@ void Debug(string sMessage, int nLevel = DEBUG_LEVEL_DEBUG, object oTarget = OBJ
             SendMessageToAllDMs(sMessage);
 
         if (nLogging & DEBUG_LOG_PC)
-            SendMessageToPC(GetFirstPC(), sMessage);
+				{
+						if(GetIsObjectValid(oTarget))

```

Comment:
The `oTarget` here is the debug target, not necessarily the PC you want to display a message for.  `oTarget` in this context is used to check for debug levels and to label debug messages on screen and in the log.  I don't think this will have the effect you're looking for, and if so, may have adverse effects on other debug logging.